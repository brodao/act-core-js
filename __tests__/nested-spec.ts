import log from '../src/logger';

test('Mensagem aninhada', () => {
	expect(log.gray('Mensagem aninhada')).toMatchSnapshot();
});
