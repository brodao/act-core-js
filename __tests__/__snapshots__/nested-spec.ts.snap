// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mensagem aninhada 1`] = `undefined`;
